src/apps/config/support/snabb-softwire-v1.lua

```diff
@@ -12,6 +13,51 @@ local ctable = require('lib.ctable')
 local cltable = require('lib.cltable')
 local path_mod = require('lib.yang.path')
 local generic = require('apps.config.support').generic_schema_config_support
+local binding_table = require("apps.lwaftr.binding_table")
```

Comment:
I think we should avoid using this module; for me it is mixing layers that shouldn't be mixed.  Instead we can just look up the IPv4 address of a softwire in the psid_map list; does that sound OK to you?